connector/src/main/java/org/geysermc/connector/network/QueryPacketHandler.java

```diff
@@ -148,7 +147,7 @@ private void sendQueryData() {
         }
 
         if (connector.getConfig().isPassthroughMotd() && pingInfo != null) {
-            String[] javaMotd = MessageUtils.getBedrockMessage(MessageSerializer.fromString(pingInfo.getDescription())).split("\n");
+            String[] javaMotd = MessageTranslator.convertMessage(pingInfo.getDescription()).split("\n");
```

Comment:
If you changed `ConnectorServerEventHandler`'s old line to `convertMessageLenient`, should you not also do that here?